x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_details/monitor_status/monitor_status_chart_theme.ts

```diff
@@ -7,24 +7,18 @@
 
 import { HeatmapStyle, RecursivePartial } from '@elastic/charts';
 import { EuiThemeComputed } from '@elastic/eui';
-import { CHART_CELL_WIDTH } from './monitor_status_data';
 
 export function getMonitorStatusChartTheme(
   euiTheme: EuiThemeComputed,
   brushable: boolean
 ): RecursivePartial<HeatmapStyle> {
   return {
     grid: {
-      cellHeight: {
-        min: 20,
-      },
```

Comment:
Unable to verify this change, @elastic/uptime could you verify this chart has no regressions from this change?

Reply:
cc @awahab07

Reply:
Looks good.